README.md

```diff
@@ -1 +1,44 @@
-# school-library-ruby
\ No newline at end of file
+# OOP School Library 
+
+> This project implements classes and objects in Ruby, Implementing encapsulation and inheritance features
+
+## Getting Started 
+
+To run this project on your machine, clone the repo 
+
+``` git clone https://github.com/mutinhiri/school-library-ruby```
+
+and open the directory ```school library ruby``
+
+you can then run ```  ruby person.rb ```
+
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@Barnabas Mutinhiri](https://github.com/mutinhiri)
+- Twitter: [@Bunbee02](https://twitter.com/Bunbee02)
+- LinkedIn: [@Bunbee](https://linkedin.com/in/linkedinhandle)
+
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] the link to the license is not working please make sure it is working as in the example provided by Microverse in [the template](https://github.com/microverseinc/readme-template).